Packs/FeedCognyteLuminar/Integrations/CognyteLuminar/CognyteLuminar.py

```diff
@@ -0,0 +1,463 @@
+API_URL = "/externalApi"
+LEAKED_RECORD = "LEAKED CREDENTIAL"
+LEAKED_CREDENTIALS_TABLE_HEADER = "Leaked Credentials from Luminar"
+IOC_TABLE_HEADER = "Indicators from Luminar"
+LUMINAR_TO_XSOAR_TYPES = {
+    'file': FeedIndicatorType.File,
+    'ipv4-addr': FeedIndicatorType.IP,
+    'ipv6-addr': FeedIndicatorType.IP,
+    'mac-addr': FeedIndicatorType.IP,
+    'domain-name': FeedIndicatorType.Domain,
+    'email-addr': FeedIndicatorType.Email,
+    'url': FeedIndicatorType.URL,
+    'windows-registry-key': FeedIndicatorType.Registry,
+    'user-account': FeedIndicatorType.Account
+}
+
+
+def enrich_incident_items(parent, childrens):
+    """
+    This will get Account, Incident objects
+    Args:
+        parent: Incident
+        childrens: Accounts
+
+    Returns: Incident, Account objects
+
+    """
+    modified_childrens = []
+
+    for children in childrens:
+        tags = [LEAKED_RECORD, f'Incident: {parent.get("name")}', f'Credentials:{children.get("credential")}']
+        additional_field = {
+            'tags': tags,
+            "category": LEAKED_RECORD,
+            'accounttype': LEAKED_RECORD,
+            'displayname': children.get('display_name'),
+            'emailaddress': children.get('display_name'),
+            'creationdate': parent.get('created'),
+            'firstseenbysource': parent.get('created'),
+            'updateddate': parent.get('modified'),
+            'lastseenbysource': parent.get('created'),
+            "shortdescription": parent.get('description') or ""
+        }
+
+        indicator = {
+            'value': children.get("account_login"),
+            'occurred': datetime.strptime(parent.get("created"),
+                                          '%Y-%m-%dT%H:%M:%S.%f%z').strftime(
+                "%m/%d/%Y, %H:%M:%S"),
+            'type': FeedIndicatorType.Account,
+            'rawJSON': dict(children),
+            'fields': additional_field,
+        }
+        modified_childrens.append(indicator)
+    return parent, modified_childrens
+
+
+def enrich_malware_items(parent, childrens):
+    """
+    This will get Malware, Indicator objects
+    Args:
+        parent: Malware
+        childrens: Indicator
+
+    Returns: Malware, Indicator Objects
+
+    """
+    modified_childrens = []
+    modified_parent = None
+    all_tags = set()
+    for children in childrens:
+        pattern = children.get("pattern")
+        for match in stix_regex_parser.findall(pattern):
+            stix_type, _, value = match
+            malware_types = parent.get("malwareTypes")
+            if stix_type in LUMINAR_TO_XSOAR_TYPES.keys():
+                indicator_type = LUMINAR_TO_XSOAR_TYPES[stix_type]
+            else:
+                indicator_type = stix_type
+            children.update(indicator_type=indicator_type)
+            parent["name"] = children["name"]
+            children["value"] = value
+            relationships_lst = EntityRelationship(
```

Comment:
can one indicator have more than one relationship? the argument name indicates so, but the actual value is only one EntityRelationship